### ISSUE TITLE
WEMI as SKOS concepts

### ISSUE THREAD

**Issue by kcoyle** [dcmi/openwemi#5]
It has been suggested that WEMI could be defined as SKOS concepts. (This may be in addition to or in place of WEMI as RDF classes.) Examples are greatly encouraged.

**Comment by kcoyle** [comment]
Here is an example taken from the [FaBiO documentation](http://www.sparontologies.net/ontologies/fabio) by @essepuntato. What is important is that fabio  classes are subclassed or sub-subclassed to one of the FRBRcore entities. (vocab.org is broken, so use the [archived version](https://web.archive.org/web/20200705103734/https://vocab.org/frbr/core) to view that vocabulary.) So WEMI comes in by inference, which is how openWEMI should be used, IMO. Obviously you can just grab a bit to do your example, and change up what you need.

`# Example of use of FaBiO #1`
`# August 18, 2015`
`# by Silvio Peroni`

`# This work is licensed under a Creative Commons Attribution 4.0`
`# International License (http://creativecommons.org/licenses/by/4.0/).`
`# You are free to:`
`# * Share - copy and redistribute the material in any medium or format`
`# * Adapt - remix, transform, and build upon the material`
`#   for any purpose, even commercially.`

`# The licensor cannot revoke these freedoms as long as you follow`
`# the license terms.`

`# Under the following terms:`
`# * Attribution - You must give appropriate credit, provide a link`
`#   to the license, and indicate if changes were made. You may do so`
`#   in any reasonable manner, but not in any way that suggests the`
`#   licensor endorses you or your use.`

@prefix : <http://www.sparontologies.net/example/> .
@prefix fabio: <http://purl.org/spar/fabio/> .
@prefix frbr: <http://purl.org/vocab/frbr/core#> .
@prefix prism: <http://prismstandard.org/namespaces/basic/2.0/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix application: <http://purl.org/NET/mediatypes/application/> .

:opjk-and-diligent a fabio:ResearchPaper ;
    dcterms:creator :casanovas , :casellas,
        :tempich, :vrandecic, :benjamins ;
    frbr:realization :version-of-record .

:version-of-record a fabio:JournalArticle ;
    dcterms:title "OPJK and DILIGENT: ontology modeling in a distributed environment" ;
    fabio:hasPublicationYear "2007"^^xsd:gYear ;
    prism:doi "10.1007/s10506-007-9036-2" ;
    frbr:embodiment :printed , :pdf ;
    frbr:partOf :ai-and-law-15-2 .

:ai-and-law-15-2 a fabio:JournalIssue ;
    prism:issueIdentifier "2" ;
    frbr:embodiment :printed-issue ;
    frbr:partOf :ai-and-law-15 .

:ai-and-law-15 a fabio:JournalVolume ;
    prism:volume "15" ;
    frbr:partOf :ai-and-law .

:ai-and-law a fabio:Journal ;
    dcterms:title "Artificial Intelligence and Law" .

:printed-issue a fabio:Paperback ;
    dcterms:publisher :springer ;
    prism:publicationDate "2007-06"^^xsd:gYearMonth ;
    frbr:part :printed .

:printed a fabio:PrintObject ;
    dcterms:publisher :springer ;
    prism:publicationDate "2007-06"^^xsd:gYearMonth ;
    prism:startingPage "171" ;
    prism:endingPage "186" .

:pdf a fabio:DigitalManifestation ;
    dcterms:publisher :springer ;
    dcterms:format application:pdf ;
    prism:publicationDate "2007-05-31"^^xsd:date .

:casanovas a foaf:Person ;
    foaf:givenName "Pompeu" ;
    foaf:familyName "Casanovas" .

:casellas a foaf:Person ;
    foaf:givenName "Nuria" ;
    foaf:familyName "Casellas" .

:tempich a foaf:Person ;
    foaf:givenName "Christoph" ;
    foaf:familyName "Tempich" .

:vrandecic a foaf:Person ;
    foaf:givenName "Denny" ;
    foaf:familyName "Vrandečić" .

:benjamins a foaf:Person ;
    foaf:givenName "Richard" ;
    foaf:familyName "Benjamins" .

:springer a foaf:Organization ;
    foaf:name "Springer" .

**Comment by tombaker** [comment]
@kcoyle The [openWEMI](https://github.com/dcmi/openwemi/blob/main/vocab/openWEMI.ttl) draft describes six classes -- four WEMI classes (Work, Expression, Manifestation, Item), all of which subclasses of Endeavor, plus ResponsibleEntity -- and four properties: relatedEndeavor, plus three "EMI" properties (expresses, manifests, and instantiates) which have as range the "WEM" classes, so I was expecting the example above to show how these properties and classes would be used in data.

Are you saying that the openWEMI classes are _not_ intended to be used directly in data, but rather as drop-in replacements for the classes of the abandoned FRBRcore vocabulary of 2005 -- specifically, to serve as superclasses for classes like ResearchPaper, JournalArticle, JournalIssue, and JournalVolume?

If so, I guess I would be looking for an example not of instance data, but of a vocabulary where the classes ResearchPaper, etc, were related (presumably as subclasses) to the openWEMI classes.

**Comment by tombaker** [comment]
I ended up providing an example of how a WEMI entity could be expressed in instance data using a SKOS concept at the end of my opening post for a new issue ["WEMI entities as data shapes?"](https://github.com/dcmi/openwemi/issues/6#issue-1297179150).

**Comment by kcoyle** [comment]
@tombaker There is no reason not to also create a skos vocabulary with WEMI, but SKOS does not allow folks to use WEMI subclasses to define domains and ranges, and that latter is what they are already doing with vocab.org's frbrcore. So this is a response to that need, and does not preclude other forms of WEMI, like SKOS or XML (which is what the akoma folks use).

**Comment by aisaac** [comment]
Yes if properties need some domain and range among the OpenWEMI entities, then mechanically they would end up as RDFS classes.
This said I have nothing against using RDFS classes as SKOS concepts too, without having to re-define new URIs. This is allowed in SKOS anyway.
